ExpressionEngine/Functions/Implementations/ConversionFunctions/FloatFunction.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Threading.Tasks;
+using ExpressionEngine.Functions.Base;
+
+namespace ExpressionEngine.Functions.Implementations.ConversionFunctions
+{
+    public class FloatFunction : Function
+    {
+        public FloatFunction() : base("float")
+        {
+        }
+
+        public override ValueTask<ValueContainer> ExecuteFunction(params ValueContainer[] parameters)
+        {
+            switch (parameters[0].Type())
```

Comment:
It would be a good idea to check the number of given parameters, to verify there's exactly one